setup.py

```diff
@@ -79,6 +79,7 @@ def get_version (mod_root):
 
         if  p.returncode   !=  0  or \
             version_detail == '@' or \
+            'git-error-while-loading' in version_detail or \
```

Comment:
maybe make that a `git-error` check, so that other git errors are also captured?

Reply:
Done